### PR TITLE
Recommend express.json() instead of body-parser

### PR DESCRIPTION
**Pull request by obrien-k**
Replace import { json } with express.json in migration, express middleware, and CORS docs. Related: https://github.com/apollographql/apollo-server/pull/7689

The code is the same but runs into fewer ESM-related troubles.

### CODE REVIEW

**Review by glasser — CHANGES_REQUESTED** [review]
I think you should look for and destroy every reference to `body-parser` in the whole repo. Specifically there are probably places in the text in the docs telling people they have to `npm install` it, which they don't!

**Comment by trevor-scheer** [comment]
Bump @obrien-k

**Comment by obrien-k** [comment]
@trevor-scheer I'll have this done by EOW, sorry for the delay.

**Comment by Meschreiber** [comment]
Just removed all other found instances of `body-parser` and replaced `bodyParser.json()` and `json()` with `express.json()`. (And some small copy edits to make instructions make sense.)

**Review by Meschreiber — APPROVED** [review]

File: docs/source/migration.mdx [hunk 1]
```diff
@@ -189,9 +189,9 @@ If you used the `apollo-server-express` package in Apollo Server 3, use the `exp
 
 To migrate from Apollo Server 3's `apollo-server-express` package to using the `expressMiddleware` function, do the following:
 
-1. Install the `@apollo/server`, `cors`, and `body-parser` packages.
+1. Install the `@apollo/server` and `cors` packages.
 2. Import symbols from `@apollo/server` (i.e., instead of from `apollo-server-express` and `apollo-server-core`).
-3. Add `cors` and `bodyParser.json()` to your server setup.
+3. Add `cors` to your server setup.
```
Comment (trevor-scheer) [inline docs/source/migration.mdx:194]: We should probably advise they update `express` to `>=4.16.0` (this is listed in their changelog as when `express.json()` was introduced)

**Review by trevor-scheer — APPROVED** [review]

File: docs/source/migration.mdx (hunk 1 above)
Comment (Meschreiber) [reply docs/source/migration.mdx:194]: Good call! https://github.com/apollographql/apollo-server/pull/7690/commits/c142fca08e98b6bdf78868a24a7aa49d6769712c

**Pull request merged**